R/helpers.R

```diff
@@ -124,17 +124,27 @@
 
 
 
-#' Pick lines from data.frame by indentifiers
+#' Pick lines from data.frame by identifiers
 #'
 #' @param x data.frame
 #' @param ... <column> = <selectionElement>
+#' @param colsMustExist logical, if \code{TRUE}, an error is thrown for missing
+#'   columns
+#' @param allowEmptyReturn logical, if \code{FALSE}, an error is thrown if the
+#'   returned data frame would have zero rows
 #' @returns data.frame with filtered rows without identifier columns
 
-.pick <- function(x, ...) {
+.pick <- function(x, ..., colsMustExist = TRUE, allowEmptyReturn = TRUE) {
```

Comment:
Not sure if it's a general R convention, but in other packages I have often seen that arguments after ```...``` often start with a ```.```
```suggestion
.pick <- function(x, ..., .colsMustExist = TRUE, .allowEmptyReturn = TRUE) {
```
I find this helpful to avoid (the arguably very unlikely) case we want to pass a column named ```colsMustExist```.